macros/edr/system/system_utils/get_config_var.sql

```diff
@@ -39,12 +39,7 @@
     'edr_cli_run': false,
     'max_int': 2147483647,
     'custom_run_started_at': none,
-    'edr_monitors': {
```

Comment:
@elongl 
The reason this was originally a var was so users could change the defaults. 
I'm not sure this is a real need, but maybe @oravi should add his input.

Reply:
It still is, I didn't change the logic here.